bot/resources/tags/windows-path.md

```diff
@@ -1,10 +1,10 @@
 **PATH on Windows**
 
-If you have installed Python but you forgot to check the *Add Python to PATH* option during the installation you may still be able to access your installation with ease.
+If you have installed Python, but you forgot to check the *Add Python to PATH* option during the installation, you may still be able to access your installation with ease.
```

Comment:
I disagree with this change. It may feel more correct but it is in-fact not: [Comma-splice - Wikipedia](https://en.wikipedia.org/wiki/Comma_splice)

To actually fix this we can make the following change:
```suggestion
If you have installed Python but forgot to check the *Add Python to PATH* option during the installation you may still be able to access your installation with ease.
```

Reply:
This isn't a comma-splice because it's not joining two separate clauses. This is a subordinate clause (iirc that's the correct grammatical term for it).

I'll remove the second "you" though since the sentence does flow better without it.

Reply:
Cool!